src/v/redpanda/application.cc

```diff
@@ -312,6 +313,17 @@ void application::initialize(
 }
 
 void application::setup_metrics() {
+    if (!config::shard_local_cfg().disable_public_metrics()) {
+        seastar::metrics::replicate_metric_families(
+          seastar::metrics::default_handle(),
+          {{"io_queue_total_read_ops", ssx::metrics::public_metrics_handle},
+           {"io_queue_total_write_ops", ssx::metrics::public_metrics_handle},
+           {"reactor_total_memory", ssx::metrics::public_metrics_handle},
```

Comment:
Is `reactor_total_memory` intended?

Reply:
No, it's not. Messed up my rebasing. Good catch.